makefile

```diff
@@ -80,6 +79,7 @@
 # MESA_INSTALL_ROOT = /opt/mesa
 # SDL_INSTALL_ROOT = /opt/sdl2
 # SDL_FRAMEWORK_PATH = $(HOME)/Library/Frameworks
+# ANDROID_SDL_HOME = /opt/sdl2-android
```

Comment:
Since you’re not automatically compiling SDL from this location anyway, why not just use the `SDL_INSTALL_ROOT` option for Android builds to specify the location of SDL?